src/Runners/PHPUnit/Options.php

```diff
@@ -820,6 +831,11 @@ public function parallelSuite(): bool
         return $this->parallelSuite;
     }
 
+    public function teamcity(): bool
+    {
+        return $this->teamcity;
```

Comment:
This needs to be tested in `\ParaTest\Tests\Unit\Runners\PHPUnit\OptionsTest`

Reply:
not relevant, tests for `--log-teamcity` exists